frontend/components/utilities/attemptLogin.js

```diff
@@ -140,14 +139,8 @@ const attemptLogin = async (
                   "mongodb+srv://${DB_USERNAME}:${DB_PASSWORD}@mongodb.net",
                   "personal",
                 ],
-                DB_USERNAME: [
-                  "user1234",
-                  "personal",
-                ],
-                DB_PASSWORD: [
-                  "ah8jak3hk8dhiu4dw7whxwe1l",
-                  "personal",
-                ],
+                DB_USERNAME: ["user1234", "personal"],
+                DB_PASSWORD: ["ah8jak3hk8dhiu4dw7whxwe1l", "personal"],
```

Comment:
I think this kind of code has to be saved in .env file

Reply:
Oh, this is just dummy data :) 
We populate this for every new users so that they have some secrets to play around with in their dashboard. It's not real secrets or environment variables.

Reply:
GitGuardian is blocking it even though it's not a real secret. Do you mind changing `ah8jak3hk8dhiu4dw7whxwe1l` to `example_password` in your branch?

This is just an example secret